app/src/main/java/com/main/mealplanner/UI/MainFragment.kt

```diff
@@ -2,22 +2,19 @@ package com.main.mealplanner.UI
 
 import android.app.usage.UsageEvents
 import android.os.Bundle
-import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.ArrayAdapter
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
-import com.main.mealplanner.MainViewModel
 import androidx.lifecycle.ViewModelProviders
-import androidx.recyclerview.widget.DefaultItemAnimator
-import androidx.recyclerview.widget.LinearLayoutManager
+import com.main.mealplanner.MainViewModel
 import com.main.mealplanner.R
 import kotlinx.android.synthetic.main.main_fragment.*
-import java.util.ArrayList
+import java.util.*
```

Comment:
For documentation reasons, it's best to import the entire package, and not use *.